hi-ml/src/health_ml/deep_learning_config.py

```diff
@@ -397,6 +397,12 @@ class TrainerParams(param.Parameterized):
         param.Number(default=None,
                      doc="PyTorch Lightning trainer flag 'limit_test_batches': Limit the test dataset to the "
                          "given number of batches if integer, or proportion of test dataset if float.")
+    pl_limit_batches: Optional[IntOrBool] = \
```

Comment:
IntOrBool is not in sync with param.Number. I'd use IntOrFloat

Reply:
but Pytorch Lightning trainer expects fast_dev_run to be Union[int, bool]. Should we maybe use just Int and set defaults to 0? a boolean is just 0 or 1 after all